web/virtual_lab/static/virtual_lab/js/physics_projectile.js

```
@@ -0,0 +1,412 @@
+    gravityValue.textContent = g.toFixed(2);
+  });
+  windSlider.addEventListener("input", () => {
+    windAccel = Number.parseFloat(windSlider.value);
+    windValue.textContent = windAccel.toFixed(2);
+  });
+
+
+  // -------- 5. Aim-by-Drag Logic -------- //
+  let isAiming = false;
+  let aimStartX = 0, aimStartY = 0;
+  let aimCurrentX = 0, aimCurrentY = 0;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)